examples/with-nextjs-and-clerk-auth/src/lib/monite-api/demo-data-generator/general.service.ts

```diff
@@ -36,11 +37,13 @@ export abstract class GeneralService {
     this.token = params.token;
     this.entityId = params.entityId;
     this.logger = params.logger;
-    this.request = createMoniteClient({
-      headers: {
-        Authorization: `${params.token.token_type} ${params.token.access_token}`,
-      },
-    });
+    this.request = createMoniteClient(params.token);
+  }
+
+  protected async getEntity(): Promise<
```

Comment:
Please, don't extend this file. If you really need `getEntity()` in the services, it would be better to use just a single shared function. These class-based generator should be refactored on pure function in the future.